airflow-core/src/airflow/triggers/deadline.py

```diff
@@ -49,7 +49,21 @@ async def run(self) -> AsyncIterator[TriggerEvent]:
 
         try:
             callback = import_string(self.callback_path)
-            result = await callback(**self.callback_kwargs)
+
+            # TODO: get airflow context
```

Comment:
Is this coming in this PR or a future PR?

Reply:
Likely a future PR